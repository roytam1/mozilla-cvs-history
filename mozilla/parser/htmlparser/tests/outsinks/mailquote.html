<html>
<head><!--
  -- The contents of this file are subject to the Netscape Public
  -- License Version 1.1 (the "License"); you may not use this file
  -- except in compliance with the License. You may obtain a copy of
  -- the License at http://www.mozilla.org/NPL/
  --  
  -- Software distributed under the License is distributed on an "AS
  -- IS" basis, WITHOUT WARRANTY OF ANY KIND, either express or
  -- implied. See the License for the specific language governing
  -- rights and limitations under the License.
  --  
  -- The Original Code is Mozilla Communicator client code, released
  -- March 31, 1998.
  -- 
  -- The Initial Developer of the Original Code is Netscape
  -- Communications Corporation. Portions created by Netscape are
  -- Copyright (C) 1998-1999 Netscape Communications Corporation. All
  -- Rights Reserved.
  -- 
  -- Contributor(s): 
  -->
<title>Ender Plain Text Test Page</title>
</head>

<body>

This page is a text of mail quoting.
<p>

<blockquote type="cite">
I hope you will enjoy this quote from <em>Hamlet</em>, introduced by a fairly long line to see how quotations get wrapped:
<p>
To be, or not to be, that is the question<br>
Whether 'tis nobler in the mind to suffer<br>
The slings and fortunes of outrageous fortune<br>
Or to take arms against a sea of troubles<br>
And by opposing end them.<br>
</blockquote>

</body>
</html>
