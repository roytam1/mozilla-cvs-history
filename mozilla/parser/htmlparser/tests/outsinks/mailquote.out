This page is a text of mail quoting.

> I hope you will enjoy this quote from Hamlet, introduced by a fairly long line to see how quotations get wrapped:
> 
> To be, or not to be, that is the question
> Whether 'tis nobler in the mind to suffer
> The slings and fortunes of outrageous fortune
> Or to take arms against a sea of troubles
> And by opposing end them.
